src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -648,18 +648,24 @@ private function addService(string $id, Definition $definition): array
         $this->referenceVariables[$id] = new Variable('instance');
 
         $return = [];
+        $returnType = '';
```

Comment:
We should revert this one: adding it provides no specific guarantee technically as this is generated code so this cannot be wrong,
and more importantly it breaks using the container to generate non-object values (yes, it's possible, and we use it in core actually)

Reply:
:+1: This has been changed to follow the changes made in the fixtures, which will be reverted.